src/test/java/ProductTest.java

```diff
@@ -28,7 +28,7 @@ public void getToken_Successful_200(){
     @BeforeMethod(onlyForGroups= "getProduct")
     public void createProduct() throws JsonProcessingException {
         Product product=new Product();
```

Comment:
it should be changed to a constructor to avoid the use of sets

Reply:
ok, I will change